games/c.yaml

```diff
@@ -173,6 +173,26 @@
   status: playable
   type: remake
   updated: 2014-07-23
+  
+- name: Cave Story Engine 2
+  type: remake
+  originals:
+  - Cave Story
+  repo: https://github.com/cuckydev/Cave-Story-Engine-2
+  development: active
+  status: playable
+  lang:
+  - C
```

Comment:
```suggestion
```
github detects header files as C but I think the project is pure C++, as shown in the makefile